tests/modules/withinheritedconstructor/point.py

```diff
@@ -1,8 +1,8 @@
-
 class Point:
-    def __init__(self, x: float=0, y: float=0):
+    def __init__(self, x: float = 0, y: float = 0):
```

Comment:
The default types here are not `float` but `int`. I'd suggest adding a `.` to make them `0.`.

Reply:
TODO :pray: